.gitignore

```diff
@@ -0,0 +1,3 @@
+/.DS_Store
+/.idea/
+/solutions/
```

Comment:
Github seems to be complaining about not having a new line at the EOF

Reply:
Yes I saw that. I see that most of the projects don't have newline at EOF for their gitignore (unlike source code files that should have EOF).